Compile fix for when profiling is off

prof_pid will be an unresolved symbol if you don't enable
profiling, and the final link of the kernel will fail. Make
the sysctl conditional on the appropriate Kconfig option.

[merge me into preempt-realtime-debug-sysctl.patch]
---
 kernel/sysctl.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/kernel/sysctl.c b/kernel/sysctl.c
index c63ef91..ace932a 100644
--- a/kernel/sysctl.c
+++ b/kernel/sysctl.c
@@ -381,6 +381,7 @@ static struct ctl_table kern_table[] = {
 		.proc_handler	= &proc_dointvec,
 	},
 #endif
+#ifdef CONFIG_PROFILING
 	{
 		.ctl_name	= CTL_UNNUMBERED,
 		.procname	= "prof_pid",
@@ -389,6 +390,7 @@ static struct ctl_table kern_table[] = {
 		.mode		= 0644,
 		.proc_handler	= &proc_dointvec,
 	},
+#endif
 #ifdef CONFIG_PREEMPT
 	{
 		.ctl_name	= CTL_UNNUMBERED,
